cli/src/pcluster/resources/imagebuilder/parallelcluster.yaml

```diff
@@ -197,7 +197,20 @@ phases:
           commands:
             - |
               set -v
+              PLATFORM='{{ build.PlatformName.outputs.stdout }}'
+
+              if [[ ${!PLATFORM} == RHEL ]]; then
+                CA_CERTS_FILE=/etc/ssl/certs/ca-bundle.crt
+              elif [[ ${!PLATFORM} == DEBIAN ]]; then
+                CA_CERTS_FILE=/etc/ssl/certs/ca-certificates.crt
+              fi
```

Comment:
Like https://github.com/aws/aws-parallelcluster-cookbook/blob/develop/cookbooks/aws-parallelcluster-install/recipes/update_certificates.rb#L18, we would like to make sure the certificates in the OS is updated by first upgrading `ca-certificates` package

Reply:
Makes sense, sorry I missed it.